server/utils/viewUtils.ts

```diff
@@ -27,4 +27,43 @@ export default class ViewUtils {
       }),
     }
   }
+
+  static summaryListArgs(summaryListItems: SummaryListItem[]): SummaryListArgs {
+    return {
+      rows: summaryListItems.map(item => {
+        return {
+          key: {
+            text: item.key,
+          },
+          value: (() => {
+            if (item.isList) {
+              const html = `<ul class="govuk-list">${item.lines
+                .map(line => `<li>${ViewUtils.escape(line)}</li>`)
```

Comment:
Would be good to test that this escaping works.

Reply:
Have added a new test for this